code/modules/jobs/job_types/roguetown/adventurer/types/pilgrim/rare/Lfish.dm

```diff
@@ -27,42 +27,102 @@
 		beltr = /obj/item/cooking/pan
 		mouth = /obj/item/rogueweapon/huntingknife
 		beltl = /obj/item/flint
-		backpack_contents = list(/obj/item/natural/worms = 2,/obj/item/rogueweapon/shovel/small=1)
+		backpack_contents = list(/obj/item/natural/worms/grub = 16,/obj/item/rogueweapon/shovel/small=1)
```

Comment:
Hey btw you can't fit 16 grubs on a satchel that only has 3 slots for things. Either change it to a normal backpack or put 2 in there.

Reply:
Lmao I thought that was a backpack, my bad let me change it to one.